solvis/solution/inversion_solution/inversion_solution_model.py

```diff
@@ -38,13 +43,19 @@ def __init__(self, solution_file: InversionSolutionFile) -> None:
 
     @property
     def solution_file(self) -> InversionSolutionFile:
+        """
+        Get the inversion solution file used by this model.
+
+        Returns:
+            InversionSolutionFile: The inversion solution file.
```

Comment:
do we need the return type in the docstring if it's provided by the type hint? https://google.github.io/styleguide/pyguide.html#doc-function-returns

Reply:
Hmm I think I forgot to specify `add docstrings in google style` here

Reply:
seems is how it thinks google docstrings are written. And this actually is better in python shell context ...

```
     |  solution_file
     |      Get the inversion solution file used by this model.
     |
     |      Returns:
     |          InversionSolutionFile: The inversion solution file.
```